assets/js/components/dashboard-splash/dashboard-splash-app.js

```diff
@@ -38,21 +38,21 @@ class DashboardSplashApp extends Component {
 	constructor( props ) {
 		super( props );
 
-		const { connectURL } = global.googlesitekit.admin;
+		const { connectURL } = global._googlesitekitLegacyData.admin;
 
 		const {
 			showModuleSetupWizard,
 			isAuthenticated,
 			isVerified,
 			hasSearchConsoleProperty,
-		} = global.googlesitekit.setup; /*eslint camelcase: 0*/
+		} = global._googlesitekitLegacyData.setup; /*eslint camelcase: 0*/
```

Comment:
Is this eslint override still necessary?

Reply:
Removed